shared/folders/files/paperkey.js

```diff
@@ -19,7 +19,8 @@ type Props = {
   checkPaperKey: (paperKey: HiddenString) => void,
 }
 
-class PaperKey extends Component<void, Props, {paperKey: string}> {
+// TODO remove this class
+class _PaperKey extends Component<void, Props, {paperKey: string}> {
```

Comment:
all of these types of 'helper' classes in connect can likely be replaced by recompose